packages/html/src/splitter/tests/splitter-splitbar-states.tsx

```diff
@@ -0,0 +1,35 @@
+import SplitterSplitbar from '../splitter-splitbar';
+import Splitter from '../splitter.spec';
+
+const style = `
+    span {
```

Comment:
Can we change this selector and use class name instead? In general, it is recommended to avoid usage of dangling element selectors